ajax/libs/vanilla-lazyload/package.json

```
@@ -1,24 +1,34 @@
 {
   "name": "vanilla-lazyload",
   "filename": "lazyload.min.js",
-  "version": "12.3.0",
-  "description": "LazyLoad is a fast, lightweight and flexible script for loading images only when they're about to enter the viewport of a scrollable area, with an excellent support to the progressive JPEG image format. Typescript module definition is available.",
+  "version": "15.1.1",
+  "description": "A fast, lightweight script to load images as they enter the viewport. SEO friendly, it supports responsive images (both srcset + sizes and picture) and progressive JPEG",
   "repository": {
     "type": "git",
     "url": "https://github.com/verlok/lazyload.git"
   },
   "license": "MIT",
-  "authors": [
-    "Andrea",
-    "verlok",
-    "Verlicchi"
-  ],
+  "author": {
+        "name": "Andrea \"verlok\" Verlicchi",
+        "email": "andrea.verlicchi@gmail.com",
+        "url": "https://www.andreaverlicchi.eu"
+    },
   "keywords": [
-    "lazy load",
-    "progressive JPEG",
-    "performance",
-    "no-jquery"
-  ],
+        "lazyload",
+        "vanilla",
+        "responsive",
+        "images",
+        "picture",
+        "srcset",
+        "SEO",
+        "intersectionObserver",
+        "sizes",
+        "progressive",
+        "performance",
+        "perfmatters",
+        "async",
+        "no-jquery"
+    ],
```

```suggestion
    "lazyload",
    "vanilla",
    "responsive",
    "images",
    "picture",
    "srcset",
    "SEO",
    "intersectionObserver",
    "sizes",
    "progressive",
    "performance",
    "perfmatters",
    "async",
    "no-jquery"
  ],
```
Please use the correct indent of two spaces.